ERCS/erc-7752.md

```diff
@@ -0,0 +1,249 @@
+        external returns (bytes32[] memory newLotIds);
+}
+```
+
+> Any function marked "external" **MUST** revert if the contract is paused, the caller/lot is frozen, or a transferController vetoes the action.
+
+### Tax Considerations (informative)
+
+The `TransferType` enum enables implementations to capture transfer context for jurisdiction-specific tax compliance:
+
+| TransferType  | Purpose                                  | Tax implications vary by jurisdiction |
+| ------------- | ---------------------------------------- | ------------------------------------- |
+| `INTERNAL`    | Administrative transfers, splits, merges | Typically no taxable event            |
+| `SALE`        | Market transactions for consideration    | Usually triggers capital gains        |
+| `GIFT`        | Gratuitous transfers                     | Donor/recipient tax treatment varies  |
+| `INHERITANCE` | Estate/probate transfers                 | Step-up rules vary by jurisdiction    |
+| `REWARD`      | Compensation, airdrops, staking rewards  | Often treated as ordinary income      |
+
+**Example U.S. Treatment**: `GIFT` uses carry-over basis (IRC § 1015), `INHERITANCE` gets stepped-up basis (IRC § 1014).  
+**Example UK Treatment**: Different rules under TCGA 1992.  
+**Example Singapore**: No capital gains tax.
+
+Implementations **SHOULD** consult local tax authorities and **MAY** implement jurisdiction-specific tax calculation modules.
+
+## Rationale
+
+**Administrative hooks (pause, freeze, forcedTransfer, adjustLot)** are embedded in ERC-7752 Core because issued assets **cannot satisfy real-world statutory and fiduciary duties without them.**
+
+- Securities law obliges issuers and registrars to rectify errors, comply with court orders, and enforce sanctions.
+- Transfer-control modules (ERC-7752-Policy) rely on a deterministic ABI to implement jurisdiction-specific rules.
+- Standardizing these events enables third-party auditors, tax engines, and regulators to ingest compliance actions uniformly.
+
+While purely decentralized assets may deem these functions unnecessary, they may be voluntarily disabled by renouncing the `ADMIN_ROLE` or setting `transferController = address(0)`. Omission from the ABI, however, would break composability for the majority of regulated implementations.
+
+## Backwards Compatibility
+
+ERC-7752 is not backward-compatible with ERC-20/721/[ERC-1155](./eip-1155.md) because each lotId represents a semi-fungible slice with its own lineage and admin controls.
+
+## Security Considerations
+
+- Implementers MUST secure admin roles (e.g., OpenZeppelin `AccessControl` or `AccessManager`).
+- A compromised `transferController` could veto or force-transfer lots; multi-sig ownership is **RECOMMENDED**.
+
+### Conclusion
+
+ERC‑7752 brings granular accounting, compliance, and administrative certainty to on‑chain representations of traditional assets—unlocking the next wave of securitised and regulated token markets.
```

Comment:
```suggestion
```

This is somewhat unnecessary, and definitely doesn't belong under the security considerations heading.